recipes/reaktoro/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+echo CC=$CC
+echo CXX=$CXX
```

Comment:
Please remove these ``echo``. Conda already do that in the beginning of the build.
also remove lines ``4`` and ``5``

Reply:
Please do the same in the build script for windows

Reply:
Oh, sure, I was debugging the Linux build (it was a work-in-progress PR), but as I said, I'll remove `build.sh`.